toolboxes/mri/pmri/gpu/cuNonCartesianSenseOperator.cu

```diff
@@ -48,11 +50,36 @@ cuNonCartesianSenseOperator<REAL,D>::mult_MH( cuNDArray< complext<REAL> >* in, c
 	  throw std::runtime_error("cuNonCartesianSenseOperator::mult_MH: input/output arrays do not match specified domain/codomains");
   }
   std::vector<size_t> tmp_dimensions = *this->get_domain_dimensions();
+  std::vector<size_t> tmp_dimensions_data = *this->get_codomain_dimensions();
+  tmp_dimensions_data.pop_back(); // Remove channel dimension 
+   if(tmp_dimensions_data[tmp_dimensions_data.size()-1]==this->ncoils_ && tmp_dimensions_data.size()>2)
+     tmp_dimensions_data.pop_back(); // High risk
```

Comment:
What are you doing here? I prefer my recons less risky - one could say "SENSEible" ;)

Reply:
Hmmmm - This was my attempt to do coil by coil gridding which is significantly faster for larger data sets. I can keep it local for now or we can discuss ways to implement it in a more sensible way :) .